ucm2/conf.d/macaudio/laptops/4spk/HiFi.conf

```diff
@@ -0,0 +1,56 @@
+# For laptops with 4 speakers (4 woofers)
+
+SectionVerb {
+	EnableSequence [
+		cset "name='Headphone Playback Mux' Primary"
+		cset "name='Speaker Playback Mux' Secondary"
+	]
+
+	Value {
+		TQ "HiFi"
+	}
+}
+
+Define.JackDevice "0"
+If.cs42l84 {
+	Condition {
+		Type ControlExists
+		Control "name='Jack ADC WNF Switch'"
+	}
+	True {
+		Include.hp.File "/conf.d/macaudio/cs42l84/Headphones.conf"
+	}
+}
+
+If.cs42l83 {
+	Condition {
+		Type ControlExists
+		Control "name='Jack ADC Boost Switch'"
+	}
+	True {
+		Include.hp.File "/conf.d/macaudio/cs42l83/Headphones.conf"
+	}
+}
+
+
+If.havespk {
+	Condition {
+		Type ControlExists
+		Control "name='Left Woofer Speaker Volume'"
+	}
+	True {
+		SectionDevice."Speaker" {
+			Comment "Speakers"
+
+			EnableSequence [
+				cset "name='Speaker Switch' on"
+			]
+
+			Value {
+				PlaybackChannels 4
```

Comment:
Can this be a variable? that would allow consolidating the 4spk and 6spk configs. We need a 2spk config for j313